src/pages/native/balance/DepositEVM.vue

```diff
@@ -87,6 +87,12 @@ export default {
             }
         },
         async generateAddress(){
+            const accountInfo = await this.$store.$api.getAccount(this.accountName);
+
+            if (accountInfo.ram_quota - accountInfo.ram_usage <= 1000){ // If account (often newly created account) does not have sufficient RAM, notify user
```

Comment:
Does the 1000 value change? can we extract it and give it a name for maintenance purposes?